ui/app/templates/components/console/log-command.hbs

```diff
@@ -1,2 +1,2 @@
 {{! using Icon here instead of Chevron because two nested tagless components results in a rendered line break between the tags breaking the layout in the <pre> }}
-<pre class="console-ui-command"><Icon @name="chevron-right" />{{@content}}</pre>
\ No newline at end of file
+<p class="console-ui-command is-font-mono"><Icon @name="chevron-right" />{{@content}}</p>
```

Comment:
`<pre>` made the command to appear on a separate line, below the icon
<img width="427" alt="Screenshot 2023-04-10 at 10 33 19 AM" src="https://user-images.githubusercontent.com/68122737/230957859-bc6dfc97-e9c2-4b7f-941b-49878e23770b.png">